src/frontend.md

```diff
@@ -289,7 +292,43 @@ export function fakeTicket() {}
 
 ## TanStack Query
 
-1. Return the full response from a request
+1. Prefer use important defaults. Read more about defaults [here](https://tanstack.com/query/v4/docs/react/guides/important-defaults) and [here](https://www.notion.so/osome/Differences-and-Features-of-cacheTime-and-staleTime-parameters-f1f622beb2144b05a4e532e0324334dd)
```

Comment:
let's list default options right here, not in external links

Reply:
that is a lot of options:)
External links is about explain important defaults and difference between staleTime and cacheTime.

Reply:
I am not sure there is a lot if we talk about only "important"
I meant would be nice to see an example of an app-wide default query client for MFs

Reply:
[This](https://github.com/OsomePteLtd/principles/pull/253/files#diff-ece74aed489b00001b4c8172bfbde28401504cdc565c6a47551d8529d8056987R311) is an example of default query client

Reply:
Agreed that listing of default might be useful
But maybe it's matter of next PR

Reply:
@strashkov @Gashaal What do you mean? Values of **all** default options we want to use?

Reply:
@Gashaal i mean the list of all default options that we create query client with, both re-declared by us and provided by react-query